### PR TITLE
MPSJ

### PR DESCRIPTION
**Pull request by madsd**

### CODE REVIEW

**Comment by seligj95** [comment]
Content looks good. Only minor feedback is that it might be worth saying there is no limit on the number of plans you can join with a subnet and that you're only limited by the number of available IPs. That's just a question that would come to mind when reading through this.

File: _posts/2023-11-14-Announcing-Multi-plan-subnet-join.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+author_name: "Mads Damgård"
+toc: true
+toc_sticky: true
+---
+
+Virtual network integration in App Service requires one subnet per App Service plan integration today. If you are working with many App Service plans, managing the subnets can be an unnecessary administrative task. Therefore, I am happy to announce that we are introducing Multi-plan subnet join (MPSJ) in limited public preview at Ignite 2023.
+
+MPSJ reduces subnet sprawl when dealing with many apps across many plans and simplifies management of networking control such as Network Security Groups and Route tables across App Service plans. With MPSJ you can join a virtual network/subnet in a different subscription, but all App Service plan joining a specific subnet must be in the same subscription.
+
+You many still want to use individual subnets if you plan to differentiate on Network Security Group configuration, NAT gateway or other subnet specific configurations.
```
Comment (TB14697) [inline _posts/2023-11-14-Announcing-Multi-plan-subnet-join.md:12]: Nit: "many" to "may"

Comment (tbhartia) [inline _posts/2023-11-14-Announcing-Multi-plan-subnet-join.md:12]: Nit pick: "many" instead of may

Comment (madsd) [reply _posts/2023-11-14-Announcing-Multi-plan-subnet-join.md:12]: Good catch

**Pull request closed without merging**